src/Domain/Creators/SphereTimeDependentMaps.hpp

```diff
@@ -142,12 +142,65 @@ struct TimeDependentMapOptions {
     std::optional<std::variant<KerrSchildFromBoyerLindquist>> initial_values{};
   };
 
+  struct RotationMapOptions {
+    using type = Options::Auto<RotationMapOptions, Options::AutoLabel::None>;
+    static std::string name() { return "RotationMap"; }
+    static constexpr Options::String help = {
+        "Options for a time-dependent rotation map about an arbitrary axis. "
+        "Specify 'None' to not use this map."};
+
+    struct InitialAngularVelocity {
+      using type = std::array<double, 3>;
+      static constexpr Options::String help = {"The initial angular velocity."};
+    };
```

Comment:
So for this, it's going to have to be a little different than the BBH domain. Instead of initial angular velocity, we'll need initial quaternion and its first two time derivatives. This is because of the `SettleToConstantQuaterion` FoT.

Reply:
I've added them as InitialValues as in the other cases